routers/user/auth.go

```diff
@@ -100,6 +100,19 @@ func SignIn(ctx *context.Context) {
 	ctx.HTML(200, tplSignIn)
 }
 
+// OpenidVerify handles response from OpenID provider
+func OpenidVerify(ctx *context.Context) {
```

Comment:
Should this be renamed to `OpenIDVerify` for consistency? It seems that most other names use `OpenID` instead of `Openid`?